libs/@blockprotocol/graph/src/shared/codegen/entity-type-meta-schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$id": "https://blockprotocol.org/type-system/0.3/schema/meta/entity-type",
+  "$id": "https://blockprotocol.org/type-system/0.3/schema/entity-type",
```

Comment:
I left this alone because this isn't actually the same schema. I had replaced it but then I realised it might not work with all of the `$ref`s and everything and we need to test it out.

We can opt to just remove the `$id` for now from this one?

Reply:
Sure

Reply:
```suggestion
```